packages/core/src/AnyClient.ts

```diff
@@ -22,4 +23,5 @@ export interface AnyClient {
   internal: InternalModelManagerNamespace;
   [$modelRelationships]?: { [modelName: string]: { [apiIdentifier: string]: { type: string; model: string } } };
   [$coreImplementation]?: AnyCoreImplementation;
+  [$args]?: typeof $args;
```

Comment:
hmm I think we don't need this on the `AnyClient` type; I don't think we'd want the key to be the symbol anyways

I think we'll either re-export this symbol from the generated client package or add it as a string property so you could do

```
await api.myVec.findFirst({select: {
  vecL2DistanceTo: {
    [api.$args]: { vector: [1,2,3] }
  }
}});
```

or something like that

Reply:
isn't that what this line enables?

Reply:
not quite as the property key would be the symbol so to access it you'd need to so

```
api[Symbol.for("gadget/fieldArgs")]
```

you could do

```
export interface AnyClient {
  $args: symbol
}
```